develop/Makefile

```
@@ -0,0 +1,25 @@
+DOCKER_NETWORK_NAME = kind
+
+-include ../.devcontainer/targets.mk
+
+LOCAL_IPV4_ADDRESS ?= $(shell hostname -I | awk '{print $$1}')
+
+dependencies lint test build dist push:
+	@echo "$@ not implemented"
+.PHONY: dependencies lint test build dist push
+
+init:
+	@./init.sh $(LOCAL_IP_ADDRESS)
+.PHONY: init
+
+local-registry:
+	docker run -d --name registry --restart=always -p 5000:5000 --net=kind registry:2
```

This will fail if the `kind` network doesn't exist yet.
```suggestion
	docker network create kind | true
	docker run -d --name registry --restart=always -p 5000:5000 --net=kind registry:2
```